ifttt/ifttt.json

```diff
@@ -49,19 +49,19 @@
           "value1": {
             "description": "First Optional Value",
             "data_type": "string",
-            "contains": ["domain", "malwr task id", "ip", "pid", "vm", "srp guid", "file size", "vault id", "firewall rule name", "carbon black query type", "port", "carbon black sensor id", "process name", "cyphort event id", "tanium question", "isightpartners report id", "flash", "pe file", "network application", "threatgrid task id", "jira project key", "rt ticket id", "sha256", "email", "host name", "carbon black query", "hash", "anubis task id", "rt queue", "javascript", "urlquery report id", "volatility profile", "os memory dump", "file path", "qradar offense id", "urlquery queue id", "carbon black watchlist", "servicenow ticket id", "md5", "mobileiron device uuid", "wepawet task id", "sha1", "lastline task id", "mac address", "url", "doc", "file name", "user name", "cuckoo task id", "jira ticket status", "pdf", "jira ticket key"],
+            "contains": [],
```

Comment:
I was using this field to allow the user to select any known value to automatically be put into this field.  For example, in the GUI playbook editor and the dropdown menu in the incidents.  

Is there a preferred way to allow this?  
Is the GUI going to be a part of your long-term vision?